source/_repos.html.slim

```diff
@@ -14,3 +14,11 @@ ul
     | For bugs with rspec-expectations:&nbsp;
     = link_to 'https://github.com/rspec/rspec-expectations', 'https://github.com/rspec/rspec-expectations'
     |  (matchers)
+  li
+    | For bugs with rspec-support:&nbsp;
+    = link_to 'https://github.com/rspec/rspec-support', 'https://github.com/rspec/rspec-support'
+    |  (string encoding, failed expectation diffing, Ruby feature detection, etc)
+  li
+    | For bugs with rspec-dev:&nbsp;
+    = link_to 'https://github.com/rspec/rspec-dev', 'https://github.com/rspec/rspec-dev'
+    |  (test runners)
```

Comment:
in the context of bug reports, though, that'd be the only thing, was my thinking, based on my experience with `bin/rubucop -> bin/rubocop` being lost in rspec-support 'cause I didn't put it in rspec-dev